lib/order.js

```diff
@@ -236,9 +246,32 @@ class Order extends Model {
    * @returns {number} finalFlags
    */
   setHidden (v) {
+    if (!v) {
+      delete this.meta?.make_visible
```

Comment:
the optional chaining operator was introduced on node 14, but this lib must support at least node 8. should we change that?

Reply:
Yes, better to support with older versions as well. Done!!